### DISCUSSION TITLE
parsing non-standard biblatex entry types

### DISCUSSION THREAD

**Discussion by bdarcus** [joostkremers/parsebib#23] [Q&A] [answered]
I noticed that parsebib parses my biblatex "jurisdiction" entry types, but that the bibtex package doesn't seem to; it returns a key not found when using `bibtex-search-entry`.

But I don't see any explicit configuration in parsebib for jurisdiction.

Any tips so that `bibtex-search-entry` sees the same thing as parsebib?

**Comment by joostkremers** [comment, marked as answer]
Parsebib just assumes anything following an `@` sign is an entry type if it's not `preamble`, `string` or `comment`. `bibtex.el` OTOH sets up a regex matching entry types in `bibtex-entry-type`, so anything that doesn't match this regex is not recognised as an entry.

`bibtex-entry-type` is populated on the basis of `bibtex-BibTeX-entry-alist` or `bibtex-biblatex-entry-alist`, depending on the detected dialect. (I don't know how `bibtex.el` detects the dialect, except that setting a file-local variable works; there may be other methods as well, I don't know.)

So in order to get `jurisdiction` recognised, you'd have to customise the entry types.

**Reply by bdarcus** [reply]
So it might be more sensible for us to use the parsebib regex to find the entry?

Edit: actually, maybe I can let-bind `bibtex-entry-type` to use the more lenient regex?
